storage/cloud-client/requester_pays_test.py

```diff
@@ -23,7 +23,14 @@
 import storage_download_file_requester_pays
 import storage_get_requester_pays_status
 
-BUCKET = os.environ["CLOUD_STORAGE_BUCKET"]
+
+# We should not use the same bucket as other tests are using.
+# First look at REQUESTER_PAYS_TEST_BUCKET and fall back to
```

Comment:
Is falling back to a default really necessary? Can't we just have an explicit check?

Reply:
That works too, but it requires us to add the environment variable in the Kokoro config. I can clean up later. Does it work for you?

Reply:
You don't need to do it as part of the kokoro config, you can add it to the [secrets tar](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/02131911100d8c9ac94e1a4b2dc18a0eb3c09f7e/.kokoro/tests/run_tests.sh#L42).

You can use the `decrypt-secrets.sh` to decrypt the current secrets, add that secret in, and use `encrypt-secrets.sh` to re-encrypt. Then just commit the new tar and the secret will be included in the run.

Reply:
+1 to Kurtis's comment - I think going for a fallback is starting to go down the rabbit hole of all of the checks we could be doing

Reply:
TIL about the secrets.tar :) Will do